prow/cmd/tot/main.go

```diff
@@ -63,6 +64,7 @@ func gatherOptions() options {
 	)
 
 	flag.StringVar(&o.configPath, "config-path", "", "Path to prow config.")
```

Comment:
Open an issue to remove this eventually for tot since it only cares about the jobs.

Reply:
you mean open an issue to remove tot eventually :-)